cc/Dockerfile

```diff
@@ -1,46 +1,49 @@
 #
 # Build stage
 #
-FROM fpco/stack-build:lts as builder
+FROM haskell as builder
 MAINTAINER Pat Brisbin <pbrisbin@gmail.com>
 
-ENV LANG en_US.UTF-8
-ENV PATH /root/.local/bin:$PATH
+ENV LANG C.UTF-8
 
 RUN mkdir -p /src
 WORKDIR /src
 
-# Initialize package/stack
+# Initialize package
 COPY hlint.cabal /src/
-RUN stack init
-RUN stack setup
+RUN cabal v2-update
 
 # Install HLint
-RUN stack install --dependencies-only
+RUN cabal v2-build --dependencies-only
 COPY ./src /src/src
 COPY ./data /src/data
 COPY ./LICENSE /src/LICENSE
-RUN stack install
+COPY ./README.md /src/README.md
```

Comment:
This part is fragile but I wanted to avoid rewriting the Dockerfile completely. Edits welcome.